content/Services/RHOBS/rules-and-alerting.md

```diff
@@ -0,0 +1,157 @@
+
+Now inside the same folder you've cloned the repo, run an instance of token-refresher:
+
+```bash
+docker run -v /token-refresher/token/:/etc/token/ --net=host quay.io/observatorium/token-refresher --oidc.client-id=<your-client-id> --oidc.client-secret=<your-client-secret> --oidc.audience=observatorium --url=https://observatorium.api.stage.openshift.com --log.level=debug --oidc.issuer-url=https://sso.redhat.com/auth/realms/redhat-external --file=/etc/token/token
+```
+
+Where you will need to provide your `--oicd.client-id` and `--oidc.client-secret` credentials. For this tutorial we will be using `https://observatorium.api.stage.openshift.com` as target URL to proxy the requests. All requests will then have the access token in the Authorization HTTP header.
+
+Now that we have set up token-refresher, let's start creating an alerting rule.
+
+### Create an alerting rule
+
+A tenant can create and list recording and alerting rules via the Observatorium Rules API. For this tutorial we will be creating an alerting rule, to also make use of the alerting capabilities that are available in Observatorium.
+
+If you want to get more details about how to interact with the Rules API and its different endpoints, refer to the [upstream documentation](https://observatorium.io/docs/design/rules-api.md/).
+
+In your local environment, create an alerting rule YAML file with the definition of the alert you want to add. Note that the file should be defined following the [Observatorium OpenAPI specification](https://github.com/observatorium/api/blob/main/rules/spec.yaml). These files should be in Prometheus [recording](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/) and [alerting](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) rules format.
+
+For example, you can create a file named `alerting-rule.yaml`:
+
+```yaml
+groups:
+- interval: 30s
+  name: test-firing-alert
+  rules:
+  - alert: TestFiringAlert
+    annotations:
+      dashboard: https://grafana.stage.devshift.net/d/Tg-mH0rizaSJDKSADX/api?orgId=1&refresh=1m
+      description: Test firing alert
+      message: Message of firing alert here
+      runbook: https://github.com/rhobs/configuration/blob/main/docs/sop/observatorium.md
+      summary: Summary of firing alert here
+    expr: vector(1)
+    for: 1m
+    labels:
+      severity: page
+      tenant_id: 0fc2b00e-201b-4c17-b9f2-19d91adc4fd2
+```
+
+Now send a `PUT` request to `/api/v1/rules/raw` endpoint, specifying the YAML file, to **create** your alerting rule using the Rules API:
+
+```bash
+curl -X PUT --data-binary @alerting-rule.yaml --header "Content-Type: application/yaml" https://observatorium.api.stage.openshift.com/api/metrics/v1/rhobs/api/v1/rules/raw
+```
+
+Besides checking if you've got a 200 response, you can also list the rules for the tenant (in this case, `rhobs`):
+
+```bash
+curl https://observatorium.api.stage.openshift.com/api/metrics/v1/rhobs/api/v1/rules/raw
+```
+
```

Comment:
I would also add a section on `How to update an alerting rule` - the detail here is that the `PUT` requests overwrite the existing data that is there so the user need to grab the existing raw rules.

Reply:
Resolving as you have added the section below.